src/settings.ts

```diff
@@ -273,6 +274,8 @@ export function load(): ISettings {
             configuration.get<string>("cwd", undefined),
         enableReferencesCodeLens:
             configuration.get<boolean>("enableReferencesCodeLens", true),
+        analyzeOpenDocumentsOnly:
+            configuration.get<boolean>("analyzeOpenDocumentsOnly", true),
```

Comment:
Don't we need a bit in `package.json` too?

Reply:
lol sure do. My b, I'll commit that in a sec. Good catch!